scripts/build.sh

```diff
@@ -22,10 +22,14 @@ source scripts/shared.sh
 
 parse_std_args "$@"
 
+aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin 137112412989.dkr.ecr.us-west-2.amazonaws.com
```

Comment:
(1) Does this work in non `us-west-2` regions? (Note there's `--region us-west-2`, as well as `us-west-2` in the image URI

(2) Could you add a link to this docs page: https://docs.aws.amazon.com/AmazonECR/latest/userguide/amazon_linux_container_image.html

Reply:
1) yeah because we only build in alpha, and other stages/regions will just pull image from alpha

2) Sure.